src/enforce/chainables/compound/index.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import runners from '../../runners';
+import { Errors } from 'Constants';
+import { runtimeError } from 'Helpers';
+
+/**
+ * Run group of tests using test runner. (e.g. `anyOf`)
+ *
+ * @private
```

Comment:
[SUPER-PETTY] Since you're exporting it, it's not really private.
Same in `chainables/single`.